.circleci/config.yml

```diff
@@ -6,32 +6,41 @@ jobs:
     executor:
       name: node/default
       tag: '10.16'
-    
+
     steps:
       - checkout
       - node/with-cache:
           steps:
             - run: npm install
       - run: npm run build
       - run: npm run test
+      - setup_remote_docker
```

Comment:
So this isn't actually the approach we want to use:
https://circleci.com/docs/2.0/building-docker-images/#separation-of-environments
this is:
https://circleci.com/docs/2.0/executor-types/#using-multiple-docker-images

Reply:
Can u pls clarify? We need redis to run in separate docker and setup remote docker does that.